Makefile

```diff
@@ -13,7 +13,7 @@ format:
 	black neuro_extras tests setup.py
 
 test_e2e:
-	pytest -vv --maxfail=3 ${PYTEST_FLAGS} tests/e2e
+	pytest -vv ${PYTEST_FLAGS} tests/e2e
```

Comment:
to be reverted

Reply:
Personally, I found it a bit frustrating to debug failing tests partially, publish the code and check other tests.. 
Mb there is a special intent for such interruption?

Reply:
well, it's e2e tests which are considered to be heavy as they run jobs etc. So, usually we abort CI tests if a few tests failed for example because of some common mistake. I agree it's not really that important.